### PR TITLE
Wip start mhdi

### PR DESCRIPTION
**Pull request by MaHDiaLaGaB**
I made a class for scraping with post scrap, time, video, and image methods, the video and image return links so I installed youtube_dl to download the video from the links, I did not finish this step tho.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        for i, j in enumerate(facebook.scrape_full_post()):
+            if facebook.where_to_start(j):
+                print(f"{i} - {j}")
+            else:
+                print("post scraped before")
+                break
+    except exceptions.NotFound:
+        print("Post, page or profile not found / doesn't exist / deleted")
+    finally:
+        print('its done')
```
Comment (Moejoe90) [inline main.py:18]: This process needs to be in scrape.py not main

File: main.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+if __name__ == '__main__':
+    grab_facebook()
```
Comment (Moejoe90) [inline main.py:22]: here we need to send the dataload of the last post reading

File: scrap.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    want to scrape
+    
+"""
+
+DELAY_SECONDS = 2
+TEST_DB = {
+    'text': 'Everything you need to know to preorder Hogwarts Legacy, the upcoming open-world RPG set in '
+            'the wizarding world of Harry Potter, hundreds of years before the hero arrived.',
+    'photo': 'https://assets-prd.ignimgs.com/2022/03/18/hogwarts-legacy-4-1647622573681.jpeg?width=1280',
+    'time': datetime.datetime(2022, 3, 24, 16, 31, 9)}
```
Comment (Moejoe90) [inline scrap.py:19]: test constants files should never be in the real code you need to load them in a separate test file

File: scrap.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        except requests.exceptions.RequestException as e:
+            raise ConnectionError(f"{self.url}: is Not reachable \nErr: {e}")
+
+    # it's static function now later with database I will make it related to this class
+    def where_to_start(self, my_dict):
+        test_dict = TEST_DB
+        if my_dict['time'] > test_dict['time']:
+            return True
+        else:
+            return False
```
Comment (Moejoe90) [inline scrap.py:47]: I don't understand this mechanism how can the function know from where it starts if its only returning true or false?

**Review by Moejoe90 — CHANGES_REQUESTED** [review]
Still some work needs to be done

File: main.py (hunk 2 above)
Comment (MaHDiaLaGaB) [reply main.py:22]: I need more information here, do you mean load just the last post or scraping information statues?

File: main.py (hunk 1 above)
Comment (MaHDiaLaGaB) [reply main.py:18]: here i am just printing, and the error exception I will remove because I did it in scrap.py

**Pull request closed without merging**